renderapps/registration/fit_transforms_by_point_match.py

```diff
@@ -98,7 +99,7 @@ def fit_transforms_by_pointmatch(render,
         # if k==1:
         #     break
 
-    return tilespecs_p
+    return tilespecs_out
```

Comment:
this seems wrong

Reply:
I don't think tilespecs_out is getting updated in any way in this code. tilespecs_p seems correct